src/lib.rs

```diff
@@ -289,9 +289,17 @@ impl SiPrefix {
 #[allow(missing_docs)]
 #[derive(Eq, PartialEq, Debug, Clone)]
 pub enum Currency {
+	/// Bitcoin mainnet
```

Comment:
Since you added docs we might just also add a short enum-level doc and remove `#[allow(missing_docs)]`.

Reply:
Makes sense!